### PR TITLE
CED-1822 EsRating follow-up TODOs

### PR DESCRIPTION
**Pull request by tomleo**
### 🔗 Linked issue

- https://energysage.atlassian.net/browse/CED-1822

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- resolving double alerts by using `@focus` instead of `@change`

### 🥼 Testing

- manual

#### 🧐 Feedback Requested / Focus Areas

- Fading the scale transition after a "click" is still an open TODO

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.
- [ ] I have documented testing approach

### CODE REVIEW

File: es-ds-docs/pages/molecules/rating.vue [hunk 1]
```diff
@@ -42,24 +38,28 @@ onMounted(async () => {
...
-                <es-rating :rating="4.5" />
-            </div>
-            <div class="bg-blue-50 justify-content-center d-flex p-200 mb-200 mb-md-0 mr-md-200">
-                <es-rating :rating="4.5" />
-            </div>
-            <div class="bg-blue-900 justify-content-center d-flex p-200 mb-200 mb-md-0 mr-md-200">
-                <es-rating :rating="4.5" />
+
+        <div class="my-500">
+            <h2>Background Variations</h2>
```
Comment (hroth1994) [inline es-ds-docs/pages/molecules/rating.vue:52]: Sentence case - Background variations

File: es-ds-components/components/es-rating.vue [hunk 2]
```diff
@@ -44,52 +44,45 @@ const props = defineProps({
...
+// Rounds to nearest .5
+const round = (value: number) => value ? Math.round(value * 2) / 2 : 0;
+const localRating = computed(() => props.rounded ? round(model.value as number) : model.value || 0);
 </script>
 
 <template>
     <div
-        v-if="readOnly"
-        :aria-label="`${roundedRating} out of 5 stars`"
+        v-if="props.readOnly"
```
Comment (hroth1994) [inline es-ds-components/components/es-rating.vue:59]: You don't need `props.` and can use the props explicitly

**Review by hroth1994 — COMMENTED** [review]
Mentioned this on Slack - the change event is still triggering twice, you just can't tell anymore without the alert. I do like the new experience in your PR, but I'm not sure removing the alert is ideal since it's a good way of observing that PrimeVue has this issue

**Comment by nathanielwarner** [comment]
@tomleo can you add the PR description and address formatting errors?

File: es-ds-docs/pages/molecules/rating.vue [hunk 3]
```diff
@@ -26,6 +20,11 @@ onMounted(async () => {
         $prism.highlight();
     }
 });
+
+const rating1Val = ref(0);
+watch(rating1Val, () => {
+    alert(rating1Val.value);
```
Comment (hroth1994) [inline es-ds-docs/pages/molecules/rating.vue:26]: Add  `// eslint-disable-next-line no-alert` to the previous line to resolve the warning

File: es-ds-components/components/es-rating.vue [hunk 4]
```diff
@@ -85,32 +53,42 @@ const update = (value: number) => {
...
+                root: {
+                    class: 'bg-transparent rounded-0 text-orange rating reactive',
+                },
+                item: (options) => {
+                    return {
+                        class: [
+                            {
+                                reactiveStar: true,
+                                focused: options.context.focused,
+                                'fade-focus': options.context.active && options.context.focused,
```
Comment (hroth1994) [inline es-ds-components/components/es-rating.vue:72]: Is this being used?

**Review by hroth1994 — COMMENTED** [review]
This is looking good, a few comments and it needs a PR description. I would include in the description what is being fixed (updates to the docs page and resolving double alerts by using @focus instead of @change) and what still needs to be changed (star transform persisting after click)

File: es-ds-components/components/es-rating.vue (hunk 4 above)
Comment (tomleo) [reply es-ds-components/components/es-rating.vue]: It was a start to getting fade to work, I'll revert it.

File: es-ds-docs/pages/molecules/rating.vue (hunk 3 above)
Comment (hroth1994) [reply es-ds-docs/pages/molecules/rating.vue:27]: I talked to Eric who suggested removing the alert and displaying the value as text underneath instead

File: es-ds-components/components/es-rating.vue [hunk 5]
```diff
@@ -130,9 +107,23 @@ const update = (value: number) => {
...
+
+    &:hover {
+        transform: scale(1.5);
+    }
+    &.focused {
+        transform: scale(1.5);
+    }
+}
+
+.rating {
```
Comment (hroth1994) [inline es-ds-components/components/es-rating.vue:121]: Is this a duplicate of the styles above, or does it serve a different purpose?

**Review by hroth1994 — COMMENTED** [review]
Looking good! A few more thoughts:

File: es-ds-docs/pages/molecules/rating.vue [hunk 6]
```diff
@@ -40,26 +36,31 @@ onMounted(async () => {
...
         <div class="my-500">
-            <h2>Form Input</h2>
+            <h2>Form input</h2>
             <es-rating
-                :read-only="false"
-                @change="changeEvent" />
-            <h2 class="mt-500">Static Display</h2>
+                v-model="rating1Val"
+                :read-only="false" />
+            <pre>Value: {{ rating1Val }}</pre>
```
Comment (hroth1994) [inline es-ds-docs/pages/molecules/rating.vue:43]: Nice! Can we add a little padding on the top?

**Review by hroth1994 — APPROVED** [review]
LGTM! 🌟

**Pull request merged**